src/material-experimental/mdc-menu/harness/menu-item-harness.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk-experimental/testing';
+import {coerceBooleanProperty} from '@angular/cdk/coercion';
+import {MenuHarnessFilters} from './menu-harness-filters';
+
+/**
+ * Harness for interacting with a standard mat-menu in tests.
+ * @dynamic
+ */
+export class MatMenuItemHarness extends ComponentHarness {
+  static hostSelector = '.mat-menu-item';
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a menu with specific attributes.
+   * @param options Options for narrowing the search:
+   *   - `label` finds a menu with specific label text.
+   * @return a `HarnessPredicate` configured with the given options.
+   */
+  static with(options: MenuHarnessFilters = {}): HarnessPredicate<MatMenuItemHarness> {
```

Comment:
Should this be MenuItemHarnessFilters? Seems weird to use the same filtering options for the overall menu and the individual items

Reply:
Done